lib/src/visitor/async_evaluate.dart

```diff
@@ -512,7 +539,62 @@ final class _EvaluateVisitor
             configuration: configuration,
             namesInErrors: true);
         _assertConfigurationIsEmpty(configuration, nameInError: true);
-      }, url: "sass:meta")
+      }, url: "sass:meta"),
+      BuiltInCallable.mixin("apply", r"$mixin, $args...", (arguments) async {
```

Comment:
Style nit: empty line above

Reply:
I agree with this, but the dart formatter does not and immediately reverts the formatting change.